releasenotes/notes/deprecate-backend-v1-props-09481526448e6bfd.yaml

```
@@ -0,0 +1,25 @@
+---
+deprecations_transpiler:
+  - |
+    The following uses of the ``BackendProperties`` object in the transpilation
+    pipeline have been deprecated as of Qiskit 1.4 and will be removed in Qiskit 2.0:
+
+    * ``backend_prop`` input argument in :class:`.DenseLayout`
+    * ``properties`` input argument in :class:`.VF2Layout`
+    * ``properties`` input argument in :class:`.VF2PostLayout`
+    * ``backend_props`` input argument in :class:`.UnitarySynthesis`
+    * ``backend_properties`` in :class:`.PassManagerConfig`
```

```suggestion
    * ``backend_props`` input argument in :class:`.UnitarySynthesis`
    * ``backend_properties`` input argument in :class:`.PassManagerConfig`
```

just for consistency - all the other `__init__`s say this.